apiserver/admin_test.go

```diff
@@ -967,9 +969,23 @@ func (s *loginSuite) TestLoginAddsAuditConversation(c *gc.C) {
 	err := conn.APICall("Admin", 3, "", "Login", request, &result)
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(result.UserInfo, gc.NotNil)
+	// Nothing's logged at this point because there haven't been any
```

Comment:
I am a bit concerned that "Login" request itself is not considered "interesting"... Most of auditing is actually VERY interested in "login" and "login" attempts :D

Reply:
Hmm. Since every client API interaction requires a login, including all logins would kind of defeat the purpose of filtering here - my `watch juju status` will still fill up the log with noise. I think you're right about logging failed login attempts though. I'll add a card for it and do it as a separate PR, if that's ok - it's pretty separate from this change.